host-support/terminal-functions.sh

```diff
@@ -97,4 +97,37 @@ check_pidevice_generation() {
             exit 1
             ;;
     esac
+}
+
+read_config() {
+    if [ -f /etc/rpi-sb-provisioner/config ]; then
+        . /etc/rpi-sb-provisioner/config
+    else
+        echo "Failed to load config. Please use configuration tool." >&2
+        exit 1
+    fi
+}
+
+get_cryptroot() {
+    if [ -f /etc/rpi-sb-provisioner/cryptroot_initramfs ]; then
+        echo "/etc/rpi-sb-provisioner/cryptroot_initramfs"
+    else
+        echo "/var/lib/rpi-sb-provisioner/cryptroot_initramfs"
+    fi
+}
+
+get_fastboot_gadget() {
+    if [ -f /etc/rpi-sb-provisioner/fastboot-gadget.img ]; then
+        echo "/etc/rpi-sb-provisioner/fastboot-gadget.img"
+    else
+        echo "/var/lib/rpi-sb-provisioner/fastboot-gadget.img"
```

Comment:
/var/lib would typically be $STATE_DIRECTORY if we wished to use systemd variables.

Reply:
I'm not convinced the indirection would be beneficial here - given that nfpm needs the fully resolved path for the install directive.